importify.cabal

```diff
@@ -16,10 +16,12 @@ library
   hs-source-dirs:      src
   exposed-modules:     Importify.Common
                        Importify.Resolution
+                       Importify.Options
```

Comment:
Usually options are put into `executable` target near `Main.hs` module. Options are not a part of library but more a part of an application.